.github/ISSUE_TEMPLATE/epic-template.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Epic
+about: This template provides a basic structure for an epic issues.
+title: '[Epic] <title>'
+labels: Epic
+---
+
+### Goal
+
+_Introduction_
+
+### Success Metrics
+
+### Work items
+
+#### Design and specification 📔
+
+- [ ] Design Document
+- [ ] Architectual Decisions Document
+- [ ] Integration Test Complete
```

Comment:
I am intrigued by this.  Can you elaborate on what you envision this looking like in the design/spec phase?